docs/Using Randomness/example-contracts.md

```diff
@@ -56,26 +55,30 @@ To use this contract, compile and deploy it in Remix.
 
 1. Open the contract in [Remix](https://remix.ethereum.org/#url=https://docs.chain.link/samples/VRF/VRFv2SubscriptionManager.sol).
 
-1. Compile and deploy the contract. Specify the constructor parameters with the desired addresses and values for your network. For a full list of available configuration variables, see the [Contract Addresses](/docs/vrf-contracts/) page.
+1. Compile and deploy the contract using the Injected Web3 environment. The contract includes all of the configuration variables that you need, but you can edit them if necessary. For a full list of available configuration variables, see the [Contract Addresses](/docs/vrf-contracts/) page.
```

Comment:
@dwightjl Shouldn't we rename the page "Contract Addresses" to "Configuration details" ?